src/main/Config.cpp

```diff
@@ -128,7 +128,7 @@ Config::Config() : NODE_SEED(SecretKey::random())
     MAXIMUM_LEDGER_CLOSETIME_DRIFT = 50;
 
     OVERLAY_PROTOCOL_MIN_VERSION = 23;
-    OVERLAY_PROTOCOL_VERSION = 26;
+    OVERLAY_PROTOCOL_VERSION = 27;
```

Comment:
with my proposed change, you don't even need to update this version

Reply:
(I guess you *could* keep this to make it easier for testing: a node with `OVERLAY_PROTOCOL_VERSION<=26` would return old style surveys)

Reply:
This changes behavior of overlay, so the version here needs to be bumped regardless (it will be 27 in 19.7.0. either way due to other changes too)

Reply:
Technically speaking it's not because the part that is really part of the overlay protocol does not include the survey response format (old nodes would forward and respond to survey requests just fine)

Reply:
Not sure I follow - if I'm talking to a node of a particular overlay version, I'm expecting a concrete protocol to be followed and a specific message format that I know about. Having the two nodes with "the same version", yet not being able to process each other's messages doesn't seem right.

Reply:
Survey is not implemented at this layer: you can receive a new format survey response via an old peer just fine.